components/pagination/src/page-controls.js

```diff
@@ -42,20 +44,25 @@ const PageControls = ({
             dataTest={`${dataTest}-page-next`}
         >
             {translate(nextPageText)}
-            <IconChevronRight16 />
+            <div className="navigator-icon">
+                <IconChevronRight16 />
+            </div>
         </Button>
         <style jsx>{`
             div {
                 display: flex;
             }
             div :global(.button-previous) {
                 height: 32px;
-                padding-left: 0;
+                padding-inline-start: 0;
             }
             div :global(.button-next) {
                 height: 32px;
-                padding-right: 0;
-                margin-left: ${spacers.dp4};
+                padding-inline-end: 0;
+                margin-inline-start: ${spacers.dp4};
+            }
+            div :global(.navigator-icon:dir(rtl)) {
+                transform: rotate(180deg);
```

Comment:
(just a `styled-jsx` tip, not a necessary change): This particular selector doesn't need to be global I don't think, since it's a plain `div` inside this JSX scope

(not part of this PR and still fine as-is, but mentioned in case it's useful, since it's related): The global `.button-previous` and `.button-next` selectors right above would be cleanest and most specific if they used the [`css.resolve`](https://github.com/vercel/styled-jsx?tab=readme-ov-file#the-resolve-tag) feature from `styled-jsx`, which would scope their styles and remove their need for global selectors -- there are a few examples of its usage around the repo, like [in here](https://github.com/dhis2/ui/blob/master/components/table/src/data-table/data-table-row/expand-handle-cell.js#L8)